src/jobs/helm-argo-rollout.yml

```
@@ -0,0 +1,211 @@
+        
+        source:
+          repoURL: << parameters.source-repo-url >>
+          targetRevision: << parameters.source-target-revision >>
+          path: << parameters.source-path >>
+          helm:
+            parameters:
+              - name: image.tag
+                value: "<< parameters.image-tag >>"
+            file: |
+              $(cat << parameters.values-file >>)
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)